content/00.build/65.developer-reference/50.era-contracts/10.l1-contracts.md

```diff
@@ -17,108 +17,202 @@ items: [{
 ---
 ::
 
-## DiamondInit
-
-It is a one-function contract that implements the logic of initializing a diamond proxy.
-It is called only once on the diamond constructor and is not saved in the diamond as a facet.
-
-Implementation detail - function returns a magic value just like it is designed in
-[EIP-1271](https://eips.ethereum.org/EIPS/eip-1271), but the magic value is 32 bytes in size.
-
-## DiamondProxy
-
-This contract uses the [EIP-2535](https://eips.ethereum.org/EIPS/eip-2535) diamond
-proxy pattern.
-
-It is an in-house implementation that is inspired by the [mudgen reference implementation](https://github.com/mudgen/Diamond).
-It has no external functions, only the fallback that delegates a call to one of the facets (target/implementation contract).
-
-So even an upgrade system is a separate facet that can be replaced.
-
-One of the differences from the reference implementation is the ability to freeze access to the facet.
+This section delves into the ZKsync Era smart contracts,
+describing how they facilitate interactions between Ethereum (Layer 1)
+and hyperchain instances (Layer 2) within ZKsync Era ecosystem.
+While we provide overview of ZKsync Era smart contracts here,
+it's important to note that this document does *NOT- cover the Hyperchain architecture -
```

Comment:
⚠️⚠️⚠️  IMPORTANT ⚠️⚠️⚠️
We need to replace all instances of the verbiage `Hyperchain` with `ZK Chain` (per legal)

Reply:
Also unsure if the `-` are intended, perhaps update to the following?
```suggestion
it's important to note that this document does *NOT* cover the ZK Chain architecture
```